streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -229,24 +229,23 @@ public boolean isRunning() {
         private final Time time;
         private final TaskManager taskManager;
         private final StreamThread streamThread;
-        private final String logPrefix;
+        private final Logger log;
 
         RebalanceListener(final Time time,
                           final TaskManager taskManager,
                           final StreamThread streamThread,
-                          final String logPrefix) {
+                          final Logger log) {
             this.time = time;
             this.taskManager = taskManager;
             this.streamThread = streamThread;
-            this.logPrefix = logPrefix;
+            this.log = log;
         }
 
         @Override
         public void onPartitionsAssigned(final Collection<TopicPartition> assignment) {
-            log.debug("{} at state {}: partitions {} assigned at the end of consumer rebalance.\n" +
+            log.debug("at state {}: partitions {} assigned at the end of consumer rebalance.\n" +
```

Comment:
Capitalize `at`

Reply:
`completed` is in lower case because before this we have applied a `logPrefix` using `LogContext`